bin/email-failed-echo-bulky-payment

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname( File::Spec->rel2abs($0) );
+    require "$d/../setenv.pl";
+}
+
+use DateTime;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use mySociety::Locale;
+
+mySociety::Locale::gettext_domain('FixMyStreet');
+
+my $comment_text = 'Echo failure email sent';
+
+my %cobrands = (
```

Comment:
As much as possible, so from here onwards ish, this should be code elsewhere (probably in CobrandEcho.pm?), so that tests can be added for it. (Then add a basic test for it.)

Reply:
Am pushing changes for your other comments in the meantime, but will get to work on this.